tuned/plugins/plugin_disk.py

```
@@ -199,43 +209,49 @@ def _instance_init_dynamic(self, instance):
 
-	def _change_spindown(self, instance, device, new_spindown_level):
-		log.debug("changing spindown to %d" % new_spindown_level)
-		(rc, out) = self._cmd.execute(["hdparm", "-S%d" % new_spindown_level, "/dev/%s" % device], no_errors = [errno.ENOENT])
-		self._update_errcnt(rc, True)
-		instance._spindown_change_delayed[device] = False
+	def _try_set_spindown(self, device, level):
+		if not self._use_hdparm:
+			return False
+		if self._hdparm_spindown_device_support.get(device) == False:
+			return False
+		(rc, out) = self._cmd.execute(["hdparm", "-S%d" % level, "/dev/%s" % device], no_errors = [errno.ENOENT])
```

Ruff (0.16.2)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)